toolkit/resources/manifests/package/toolchain_aarch64.txt

```diff
@@ -23,6 +23,8 @@ ca-certificates-base-2.0.0-9.cm2.noarch.rpm
 ca-certificates-legacy-2.0.0-9.cm2.noarch.rpm
 ca-certificates-shared-2.0.0-9.cm2.noarch.rpm
 ca-certificates-tools-2.0.0-9.cm2.noarch.rpm
+ccache-4.6-2.cm2.aarch64.rpm
```

Comment:
Shouldn't we also add it to the `pkggen_core_*txt` manifests, so we have `ccache` present in the chroot?